src/styles/_mixins.scss

```diff
@@ -5,6 +5,44 @@
     @return #{$remSize}rem;
 }
 
+@mixin md-icon {
+    -webkit-font-feature-settings: 'liga';
+    -webkit-font-smoothing: antialiased;
+    direction: ltr;
+    display: inline-block;
+    font-family: 'Material Icons';
+    font-size: 24px;
```

Comment:
`$font-size-xx-large`

Reply:
It could be problematic, because we might want to change the size of that font size but not the icons